twingate/client_connector_test.go

```diff
@@ -464,3 +363,65 @@ func TestClientConnectorReadAllOk(t *testing.T) {
 		assert.EqualValues(t, len(mockMap), counter)
 	})
 }
+
+func TestClientConnectorUpdateRequestError(t *testing.T) {
+	t.Run("Test Twingate Resource : Client Connector Update Request Error", func(t *testing.T) {
+
+		client := newHTTPMockClient()
+		defer httpmock.DeactivateAndReset()
+		httpmock.RegisterResponder("POST", client.GraphqlServerURL,
+			httpmock.NewErrorResponder(errors.New("error_1")))
+		connectorId := graphql.ID("test")
+
+		err := client.updateConnector(context.Background(), connectorId, "new name")
+
+		assert.EqualError(t, err, "failed to update connector with id test: Post \"https://test.twindev.com/api/graphql/\": error_1")
```

Comment:
Can't figure out why is expecting this specific URL `test.twindev.com` and where it is configured

Reply:
We set up everything at ` newHTTPMockClient` - https://github.com/Twingate/terraform-provider-twingate/blob/37fd097212777e9e23da5db472464086f2a1b87c/twingate/client_test.go#L10